src/index.js

```diff
@@ -1,7 +1,7 @@
 /*eslint-disable*/
```

Comment:
- [ ] Implementing linters in your project will help ensure that the quality of your written code is good. Therefore, I suggest you don't disable linter checking, but rather find a way to improve your code.